orderbook/src/api/get_user_orders.rs

```diff
@@ -0,0 +1,103 @@
+use crate::{api::internal_error, orderbook::Orderbook};
+use anyhow::Result;
+use model::order::Order;
+use serde::Deserialize;
+use shared::H160Wrapper;
+use std::{convert::Infallible, sync::Arc};
+use warp::{
+    hyper::StatusCode,
+    reply::{self, with_status, Json, WithStatus},
+    Filter, Rejection, Reply,
+};
+
+#[derive(Clone, Copy, Debug, Deserialize)]
+struct Query {
+    offset: Option<u64>,
+    limit: Option<u64>,
```

Comment:
Maybe we should enforce `limit` to be non-zero (as fetching 0 orders doesn't really make sense):
```suggestion
    limit: Option<NonZeroU64>,
```

Just a suggestion, and not sure we actually need to, feel free to disregard.

Reply:
Added check for this. Was in fact needed because a limit 0 would be unlimited again in Postgres.